Cargo.toml

```diff
@@ -35,7 +35,7 @@ debug = true
 
 [dependencies]
 fixedbitset = { version = "0.4.0", default-features = false }
-indexmap = "2.0"
+indexmap = "~2.5.0"
```

Comment:
Please don't do this! It prevents users with more current compilers from using the latest petgraph and indexmap together, even though they would otherwise work fine.

A better way is to downgrade it just for your MSRV CI, either manually with `cargo update --precise` or using Rust 1.84's new resolver (in beta) before running the test with your actual MSRV.

* https://doc.rust-lang.org/beta/cargo/reference/resolver.html#rust-version
* to be announced: https://github.com/rust-lang/blog.rust-lang.org/pull/1451

Reply:
I also just hit this, since I have another dependency that specifies `indexmap = "2.7"`.